### ISSUE TITLE
Ambiguous Labelling Convention and Button States

### ISSUE THREAD

**Issue by NDo-UW** [wordplaydev/wordplay#840]
### What are you trying to do that you can't?

While learning how to use Wordplay, I was confused on the labelling system of the window buttons in the navigation, specifically the "start" button. When looking to open the code editor, it was not intuitive for me to select the "start" button. In addition to the window naming system, there’s minimal visual distinction between active/inactive buttons making it difficult to know which button was active or inactive.

### What is your idea?

To solve the labelling convention for the "start" button, I would make the label suggest that it would open a code editor such as "Editor" or "Code Editor".

Additionally, to solve the issue of the button states, I think that the normal and active button states should be more distinguishable from each other through the use of color contrast and visual feedback. For example, the active buttons could have a highlighted background color or accent border.

**Comment by amyjko** [comment]
Thanks for reporting Nathan! I think these two issues are reasonably close enough to be considered one.

For the future: a screenshot would be helpful to reduce any ambiguity about which specific user interface elements you're referring to. I'm *think* you're referring to the source file toggle, which shows the source file's name (which defaults to "start"), correct?

**Comment by NDo-UW** [comment]
Yes, that is correct! I'll attach a screen shot here.

<img width="687" height="49" alt="Image" src="https://github.com/user-attachments/assets/98aa67b9-21d5-4c97-bf2f-70a0ab5344b4" />

**Comment by amyjko** [comment]
@NDo-UW I pushed some minor revisions to these, including a slight visual redesign, and a default label of "code" for when there is a single source file, but showing the source file's names if there are multiple. Let me know if you feel it resolves this issue. If it doesn't, feel free to comment and suggest we reopen and iterate.
